js/build-rss-feed.js

```diff
@@ -421,6 +421,10 @@ var rss = (function() {
   return rss;
 })();
 
+Fliplet().then(function() {
```

Comment:
Can you check if the `Fliplet()` is necessary? The `build.js` file runs `$container.translate()` after `Fliplet()` has resolved, which means `Fliplet()` isn't required here.

Also, if `build.js` is running `$container.translate()` why is it necessary to run `$container.translate()` again in `build-rss-feed.js`?